<launch>
  <node pkg="pr2_mechanism_control" type="spawner.py" args="$(find life_test)/laser_tilt_test/life_test/controllers.xml" />
  <node pkg="pr2_mechanism_controllers" type="send_periodic_cmd_srv.py" args="laser_tilt_controller linear 0.25 1.25 0.25" />
  <!-- node pkg="life_test" type="run_laser_tilt_test.py" / -->
</launch>
